application/lib/view/screens/TasksScreen.dart

```diff
@@ -0,0 +1,275 @@
+                flex: 33,
+                child: Container(
+                  alignment: Alignment.topCenter,
+                  child: SingleChildScrollView(
+                    child: Column(
+                      children: doneWidgets,
+                    ),
+                  ),
+                  color: Colors.lightGreen,
+                )),
+          ],
+        )
+      ],
+    );
+  }
+
+  Widget generateAppBarLandscape() {
+    return AppBar(
+      centerTitle: true,
+      title: Text(this.app.appName + " - Select Task"),
+      leading: IconButton(
+        icon: FaIcon(FontAwesomeIcons.arrowLeft),
+        onPressed: () => Routes.pop(this.contextObject.getOutput()),
+      ),
+      actions: <Widget>[
+        createNewTaskButton(),
+      ],
+    );
+  }
+
+  Widget _columnHeader() {
+    return Row(children: <Widget>[
+      Expanded(
+          flex: 33,
+          child: Container(
+            decoration: BoxDecoration(border: Border.all()),
+            alignment: Alignment.topCenter,
+            child: Column(
+              children: <Widget>[
+                FaIcon(FontAwesomeIcons.hourglass),
+                Text(
+                  "Not started",
+                  style: TextStyle(fontSize: 16),
+                ),
+              ],
+            ),
+          )),
+      Expanded(
+          flex: 33,
+          child: Container(
+            decoration: BoxDecoration(border: Border.all()),
+            alignment: Alignment.topCenter,
+            child: Column(
+              children: <Widget>[
+                FaIcon(FontAwesomeIcons.cog),
+                Text(
+                  "WIP",
+                  style: TextStyle(fontSize: 16),
+                ),
+              ],
+            ),
+          )),
+      Expanded(
+          flex: 33,
+          child: Container(
+            decoration: BoxDecoration(border: Border.all()),
+            alignment: Alignment.topCenter,
+            child: Column(
+              children: <Widget>[
+                FaIcon(FontAwesomeIcons.check),
+                Text(
+                  "Done",
+                  style: TextStyle(fontSize: 16),
+                ),
+              ],
+            ),
+          )),
+    ]);
+  }
+
+  Widget generateAppBarPortrait() {
+    return AppBar(
+      centerTitle: true,
+      title: Text(this.app.appName + " - Select Task"),
+      leading: IconButton(
+        icon: FaIcon(FontAwesomeIcons.arrowLeft),
+        onPressed: () => Routes.pop(this.contextObject.getOutput()),
+      ),
+      actions: <Widget>[
+        createNewTaskButton(),
+      ],
+      bottom: TabBar(
+        tabs: [
+          Tab(
+              icon: Column(
+            children: <Widget>[
+              FaIcon(FontAwesomeIcons.hourglass),
+              Text("Not started"),
+            ],
+          )),
+          Tab(
+              icon: Column(
+            children: <Widget>[
+              FaIcon(FontAwesomeIcons.cog),
+              Text("WIP"),
```

Comment:
```suggestion
              Text("Work in Progress"),
```

Reply:
Samme som ovenfor? 
Der kan ikke stå Work In Progress på telefon skærme, men vi kan lave så der står WIP i portrait og Work In Progress i landscape